docs/integration-handbook.md

```diff
@@ -0,0 +1,246 @@
+The lower limit specifies the vertical angle between the horizontal plane of the lidar and the most bottom laser. 
+The upper limit specifies the vertical angle between the horizontal plane of the lidar and most upper laser. 
+
+The horizontal field of view of the lidar is specified with an upper and lower limit in degree as well. Only points within this FOV will be returned. 
+The lower limit specifies the counterclockwise angle on a top view from the direction the lidar is pointing towards. 
+The upper limit specifies the clockwise angle on a top view from the direction the lidar is pointing towards. 
+
+For every lidar, the rotation speed (Hz) and capture frequency (Hz) must be chosen. 
+The rotation speed specifies how fast the lasers spin and the capture frequency specifies how often a point cloud is created.
+
+While rotating, only lasers within the horizontal field of view are captured. 
+
+> For example, a lidar with 190 degrees horizontal field of view, rotating at 10hz with a capture frequency of 20 Hz will receive point clouds covering anything from 10 to 180 degrees of the field of view.
+
+There is no guarantee that the rotation speed and capture frequency stay in sync. 
+You won’t be able to rely on synchronization of rotation speed and capture frequency. 
+A lidar rotating at 5 Hz would theoretically have rotated 50 times after 10 seconds but in reality, this will be somewhere between 45 and 55 times. 
+
+For every lidar, you can specify the resolution: the total number of points collected if the lasers would do a 360 field of view sweep scan. 
+This value is used to calculate the number of points in each laser and the spacing between the points. 
+
+Every lidar capture is limited to collecting 10000 points. 
+The maximum number of points collected during a capture is calculated by dividing the lidar’s resolution by the horizontal field of view fraction.
+
+> For example, a lidar with a 30 degrees horizontal field (horizontal FOV fraction of 30/360 = 1/12) can have a maximum resolution of 120000.
+
+The total number of points per second is limited to 100000 points.
+
+> For example, a first lidar collects 10000 points per capture at 5 hz, a second lidar collects 8000 points per capture at 5 hz. 
+  This is valid because in total they collect 90000 points per second.
+
+### GPS
+Every vehicle has 1 GPS, it is located at the centre of gravity of the vehicle.
+This sensor cannot be removed or moved.
+
+The GPS captures the position of the vehicle in the geodetic reference system, namely longitude [deg], latitude [deg], altitude [m].
+More detailed technical information about the accuracy of the GPS can be found [here](gps.md).
+
+### IMU
+Every vehicle has 1 IMU, it is located at the centre of gravity of the vehicle.
+This sensor cannot be removed or moved.
+
+The IMU captures the acceleration, orientation and angular rate of the vehicle in the ?? frame.
```

Comment:
??
Are you referring to the current simulation frame?

Reply:
oops I left it like this, it should be the vehicle center of gravity frame which is called /odom right @SijmenHuizenga ?

Reply:
Yes! It captures it in the `fsds/FSCar` frame. This frame centers the center of the car. The center of the car is the Unreal Engine car pawn position, which in turn is also the center of gravity. Maybe we can link to ros-bridge.md for further reading on frames.

Reply:
Soo I have combined the list of topics details from the handbook into the `ros-bridge`. Now there is only 1 place where all technical details of the topics are described and the handbook just shows the topic names and what to functionally do with those.